ddtrace/context.py

```diff
@@ -22,7 +22,7 @@ class Context(object):
 
     This data structure is thread-safe.
     """
-    def __init__(self, trace_id=None, span_id=None, sampled=True, sampling_priority=None):
+    def __init__(self, trace_id=None, span_id=None, sampled=True, sampling_priority=None, service=None):
```

Comment:
Since I don't want to expose at the moment this API publicly, can we change the kwarg as `_service=None`? We can then use it to address this specific issue, but will be safe for us to break it in the future, if it's needed while supporting other standards such as OpenTracing (they have a `SpanContext` too).

Reply:
done

Reply:
removed so marking resolved